ktor-server/ktor-server-plugins/ktor-server-content-negotiation/jvm/src/io/ktor/server/plugins/contentnegotiation/ContentNegotiationConfig.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.plugins.contentnegotiation
+
+import io.ktor.http.*
+import io.ktor.serialization.*
+import kotlin.reflect.*
+
+/**
+ * Configuration type for [ContentNegotiation] plugin
+ */
+public class ContentNegotiationConfig : Configuration {
```

Comment:
may be not the right place, but: In all ktor codebase (specifically plugins) there is 2 namings for plugins configuration: *Config and *Configuration. Even in plugins API `createRouteScopedPlugin` has parameter called `createConfiguration` but has type parameter called `PluginConfigT`, what's more, it's create an instance of `RouteScopedPlugin<PluginConfigT>` which has declaration RouteScopedPlugin<TConfiguration>.

Looks like, this should be somehow generalized, so renamed, to always using or `*Config` or `*Configuration` namings.

Other thing: here `ContentNegotiationConfig` extends `Configuration` - for me it's a little confusing.

And the last thing, may be it's better to rename `Configuration` from `ktor-serialization` to something more specific? like `ContentConverterConfiguration` or anything else. Name is too short and widely used.

Reply:
Hey @olme04, thank you for the notice! We agree to rename `Configuration` to `Config` everywhere and put prefixes with a clear name.

Reply:
ah, several more questions why we are here:)
* what is the reason of naming of shared module as `ktor-serialization` but client/server plugins as  `ktor-server/client-content-negotiation` and the only thing exists inside `ktor-serialization` is `ContentConverter`s related declarations?
* why not to use or `content-negotiation` or `serialization` or other consistent naming everywhere? 
* may be it's better to merge `ktor-serialization` and `ktor-websocket-serialization` modules, as `WebsocketContentConverter` and dependency on `ktor-websockets` is already inside `ktor-serialization`?
* if not, one more proposal for naming alignment would be to use in modules names or `websocket` or `websockets` :)

Reply:
@e5l ping